apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/gradingPanel.html.ts

```diff
@@ -92,6 +96,51 @@ export function GradingPanel({
               </li>
             `
           : ''}
+        ${showAiSubmissionGroup && context === 'main'
```

Comment:
It confuses me that AI submission grouping is controlled by the AI grading mode toggle. I feel like the toggle might need a better name?

Reply:
I like both of those names.

Reply:
We discussed that this should happen in a follow-up PR.